GroupIS1024/Models/Team.cs

```diff
@@ -1,18 +1,20 @@
 ﻿using System.ComponentModel.DataAnnotations;
 using System.ComponentModel;
+using Newtonsoft.Json;
 
 namespace GroupIS1024.Models
 {
     public class Team
     {
-
+        [JsonProperty("Team Id")]
         public int TeamId { get; set; }
 
+        [JsonProperty("Team Name")]
         public string TeamName { get; set; }
 
-        public List<Coach> Coaches { get; set; }
+        public List<Coach>? Coaches { get; set; }
```

Comment:
Given the previous models. It is impossible to create either team, player, or coach first. All are interdependent on each other to be created. So, I have made foreign key properties nullable to resolve the chicken and egg problem